diff --git a/browser/app/profile/firefox.js b/browser/app/profile/firefox.js
index 56c8022ec4bbeda70e09c3b8780afcbf21ba0579..c2d6a643ae8673d8aec3b648f3c525efe17223d4 100644
--- a/browser/app/profile/firefox.js
+++ b/browser/app/profile/firefox.js
@@ -33,7 +33,7 @@ pref("extensions.checkCompatibility.temporaryThemeOverride_minAppVersion", "29.0
 
 pref("extensions.webextOptionalPermissionPrompts", true);
 // If enabled, install origin permission verification happens after addons are downloaded.
-pref("extensions.postDownloadThirdPartyPrompt", true);
+pref("extensions.postDownloadThirdPartyPrompt", false);
 
 // Preferences for AMO integration
 pref("extensions.getAddons.cache.enabled", true);
@@ -218,16 +218,20 @@ pref("lightweightThemes.getMoreURL", "https://addons.mozilla.org/%LOCALE%/firefo
 #endif
 
 // UI tour experience.
-pref("browser.uitour.enabled", true);
+pref("browser.uitour.enabled", false);
 pref("browser.uitour.loglevel", "Error");
 pref("browser.uitour.requireSecure", true);
 pref("browser.uitour.themeOrigin", "https://addons.mozilla.org/%LOCALE%/firefox/themes/");
 pref("browser.uitour.url", "https://www.mozilla.org/%LOCALE%/firefox/%VERSION%/tour/");
 // How long to show a Hearbeat survey (two hours, in seconds)
 pref("browser.uitour.surveyDuration", 7200);
-
+pref("browser.fixup.alternate.enabled", false);
+pref("devtools.debugger.remote-enabled", false);
 pref("keyword.enabled", true);
+pref("middlemouse.contentLoadURL", false);
+pref("webgl.disabled", false);
 
+pref("ui.systemUsesDarkTheme", 1);
 // Fixup whitelists, the urlbar won't try to search for these words, but will
 // instead consider them valid TLDs. Don't check these directly, use
 // Services.uriFixup.isDomainKnown() instead.
@@ -263,7 +267,7 @@ pref("browser.compactmode.show", false);
 
 // At startup, check if we're the default browser and prompt user if not.
 pref("browser.shell.checkDefaultBrowser", true);
-pref("browser.shell.shortcutFavicons",true);
+pref("browser.shell.shortcutFavicons",false);
 pref("browser.shell.mostRecentDateSetAsDefault", "");
 pref("browser.shell.skipDefaultBrowserCheckOnFirstRun", true);
 pref("browser.shell.didSkipDefaultBrowserCheckOnFirstRun", false);
@@ -281,12 +285,17 @@ pref("browser.shell.setDefaultPDFHandler", true);
 // as Acrobat Reader or Nitro PDF.
 pref("browser.shell.setDefaultPDFHandler.onlyReplaceBrowsers", true);
 #endif
+pref("security.ssl.require_safe_negotiation", true);
+pref("security.tls.enable_0rtt_data", false);
+pref("security.OCSP.enabled", 1);
+pref("security.OCSP.require", true);
+pref("security.family_safety.mode", 0);
 
 
 // 0 = blank, 1 = home (browser.startup.homepage), 2 = last visited page, 3 = resume previous browser session
 // The behavior of option 3 is detailed at: http://wiki.mozilla.org/Session_Restore
-pref("browser.startup.page",                1);
-pref("browser.startup.homepage",            "about:home");
+pref("browser.startup.page",                3);
+pref("browser.startup.homepage",            "about:blank");
 #ifdef NIGHTLY_BUILD
 pref("browser.startup.homepage.abouthome_cache.enabled", true);
 #else
@@ -360,7 +369,6 @@ pref("browser.urlbar.ctrlCanonizesURLs", true);
 // Whether we announce to screen readers when tab-to-search results are
 // inserted.
 pref("browser.urlbar.accessibility.tabToSearch.announceResults", true);
-
 // Control autoFill behavior
 pref("browser.urlbar.autoFill", true);
 
@@ -372,7 +380,7 @@ pref("browser.urlbar.autoFill.adaptiveHistory.enabled", false);
 pref("browser.urlbar.autoFill.adaptiveHistory.minCharsThreshold", 0);
 
 // Whether to warm up network connections for autofill or search results.
-pref("browser.urlbar.speculativeConnect.enabled", true);
+pref("browser.urlbar.speculativeConnect.enabled", false);
 
 // Whether bookmarklets should be filtered out of Address Bar matches.
 // This is enabled for security reasons, when true it is still possible to
@@ -391,10 +399,11 @@ pref("browser.urlbar.maxHistoricalSearchSuggestions", 2);
 // The default behavior for the urlbar can be configured to use any combination
 // of the match filters with each additional filter adding more results (union).
 pref("browser.urlbar.suggest.bookmark",             true);
+pref("browser.search.suggest.enabled", false);
 pref("browser.urlbar.suggest.history",              true);
 pref("browser.urlbar.suggest.openpage",             true);
 pref("browser.urlbar.suggest.remotetab",            true);
-pref("browser.urlbar.suggest.searches",             true);
+pref("browser.urlbar.suggest.searches",             false);
 pref("browser.urlbar.suggest.topsites",             true);
 pref("browser.urlbar.suggest.engines",              true);
 pref("browser.urlbar.suggest.calculator",           false);
@@ -425,13 +434,13 @@ pref("browser.urlbar.suggest.bestmatch", true, sticky);
 // pref is exposed to the user in the UI, and it's sticky so that its
 // user-branch value persists regardless of whatever Firefox Suggest scenarios,
 // with their various default-branch values, the user is enrolled in over time.
-pref("browser.urlbar.suggest.quicksuggest.nonsponsored", false, sticky);
+pref("browser.urlbar.suggest.quicksuggest.nonsponsored", false);
 
 // Whether sponsored quick suggest results are shown in the urlbar. This pref is
 // exposed to the user in the UI, and it's sticky so that its user-branch value
 // persists regardless of whatever Firefox Suggest scenarios, with their various
 // default-branch values, the user is enrolled in over time.
-pref("browser.urlbar.suggest.quicksuggest.sponsored", false, sticky);
+pref("browser.urlbar.suggest.quicksuggest.sponsored", false);
 
 // Whether data collection is enabled for quick suggest results in the urlbar.
 // This pref is exposed to the user in the UI, and it's sticky so that its
@@ -500,6 +509,10 @@ pref("browser.urlbar.switchTabs.adoptIntoActiveWindow", false);
 // Whether addresses and search results typed into the address bar
 // should be opened in new tabs by default.
 pref("browser.urlbar.openintab", false);
+pref("network.cookie.lifetimePolicy", 2);
+pref("media.gmp-widevinecdm.visible", true);
+pref("media.gmp-widevinecdm.enabled", true);
+
 
 // Enable three-dot options button and menu for eligible results.
 pref("browser.urlbar.resultMenu", true);
@@ -522,7 +535,7 @@ pref("browser.urlbar.showSearchTerms.featureGate", false);
 
 // If true, show the search term in the Urlbar while on
 // a default search engine results page.
-pref("browser.urlbar.showSearchTerms.enabled", true);
+pref("browser.urlbar.showSearchTerms.enabled", false);
 
 // Controls the empty search behavior in Search Mode:
 //  0 - Show nothing
@@ -597,10 +610,12 @@ pref("browser.download.loglevel", "Error");
 // feedback from their action.
 pref("browser.download.saveLinkAsFilenameTimeout", 4000);
 
-pref("browser.download.useDownloadDir", true);
+pref("browser.download.useDownloadDir", false);
 pref("browser.download.folderList", 1);
-pref("browser.download.manager.addToRecentDocs", true);
+pref("browser.download.manager.addToRecentDocs", false);
 pref("browser.download.manager.resumeOnWakeDelay", 10000);
+pref("browser.download.always_ask_before_handling_new_types", true);
+pref("extensions.enabledScopes", 5);
 
 // This records whether or not the panel has been shown at least once.
 pref("browser.download.panel.shown", false);
@@ -623,7 +638,7 @@ pref("browser.download.autohideButton", true);
 
 // Controls whether to open the downloads panel every time a download begins.
 // The first download ever run in a new profile will still open the panel.
-pref("browser.download.alwaysOpenPanel", true);
+pref("browser.download.alwaysOpenPanel", false);
 
 // Determines the behavior of the "Delete" item in the downloads context menu.
 // Valid values are 0, 1, and 2.
@@ -632,9 +647,7 @@ pref("browser.download.alwaysOpenPanel", true);
 //   2 - Remove the download from both session list and history.
 pref("browser.download.clearHistoryOnDelete", 0);
 
-#ifndef XP_MACOSX
-  pref("browser.helperApps.deleteTempFileOnExit", true);
-#endif
+pref("browser.helperApps.deleteTempFileOnExit", true);
 
 // This controls the visibility of the radio button in the
 // Unknown Content Type (Helper App) dialog that will open
@@ -660,7 +673,7 @@ pref("browser.search.context.loadInBackground", false);
 pref("browser.search.hiddenOneOffs", "");
 
 // Mirrors whether the search-container widget is in the navigation toolbar.
-pref("browser.search.widget.inNavBar", false);
+pref("browser.search.widget.inNavBar", true);
 
 // Enables display of the options for the user using a separate default search
 // engine in private browsing mode.
@@ -697,7 +710,7 @@ pref("browser.privacySegmentation.preferences.show", false);
 pref("browser.sessionhistory.max_entries", 50);
 
 // Built-in default permissions.
-pref("permissions.manager.defaultsUrl", "resource://app/defaults/permissions");
+pref("permissions.manager.defaultsUrl", "");
 
 // Set default fallback values for site permissions we want
 // the user to be able to globally change.
@@ -725,7 +738,9 @@ pref("browser.link.open_newwindow.override.external", -1);
 // 0: no restrictions - divert everything
 // 1: don't divert window.open at all
 // 2: don't divert window.open with features
-pref("browser.link.open_newwindow.restriction", 2);
+pref("browser.link.open_newwindow.restriction", 0);
+pref("browser.startup.homepage_override.mstone", "ignore");
+
 
 // If true, this pref causes windows opened by window.open to be forced into new
 // tabs (rather than potentially opening separate windows, depending on
@@ -889,10 +904,10 @@ pref("browser.bookmarks.defaultLocation", "toolbar");
 pref("dom.disable_open_during_load",              true);
 
 // allow JS to move and resize existing windows
-pref("dom.disable_window_move_resize",            false);
+pref("dom.disable_window_move_resize",            true);
 // prevent JS from monkeying with window focus, etc
 pref("dom.disable_window_flip",                   true);
-
+pref("accessibility.force_disabled", 1);
 // popups.policy 1=allow,2=reject
 pref("privacy.popups.policy",               1);
 pref("privacy.popups.usecustom",            true);
@@ -900,13 +915,13 @@ pref("privacy.popups.showBrowserMessage",   true);
 
 pref("privacy.item.cookies",                false);
 
-pref("privacy.clearOnShutdown.history",     true);
+pref("privacy.clearOnShutdown.history",     false);
 pref("privacy.clearOnShutdown.formdata",    true);
 pref("privacy.clearOnShutdown.downloads",   true);
 pref("privacy.clearOnShutdown.cookies",     true);
 pref("privacy.clearOnShutdown.cache",       true);
 pref("privacy.clearOnShutdown.sessions",    true);
-pref("privacy.clearOnShutdown.offlineApps", false);
+pref("privacy.clearOnShutdown.offlineApps", true);
 pref("privacy.clearOnShutdown.siteSettings", false);
 pref("privacy.clearOnShutdown.openWindows", false);
 
@@ -914,7 +929,7 @@ pref("privacy.cpd.history",                 true);
 pref("privacy.cpd.formdata",                true);
 pref("privacy.cpd.passwords",               false);
 pref("privacy.cpd.downloads",               true);
-pref("privacy.cpd.cookies",                 true);
+pref("privacy.cpd.cookies",                 false);
 pref("privacy.cpd.cache",                   true);
 pref("privacy.cpd.sessions",                true);
 pref("privacy.cpd.offlineApps",             false);
@@ -931,7 +946,7 @@ pref("privacy.history.custom",              false);
 // 4 - Today
 // 5 - Last 5 minutes
 // 6 - Last 24 hours
-pref("privacy.sanitize.timeSpan", 1);
+pref("privacy.sanitize.timeSpan", 0);
 
 pref("privacy.sanitize.migrateFx3Prefs",    false);
 
@@ -1005,11 +1020,25 @@ pref("browser.history_swipe_animation.disabled", false);
 
 pref("mousewheel.with_win.action", 1);
 
-pref("browser.xul.error_pages.expert_bad_cert", false);
+pref("browser.xul.error_pages.expert_bad_cert", true);
 pref("browser.xul.error_pages.show_safe_browsing_details_on_load", false);
+pref("network.http.referer.XOriginPolicy", 2);
+pref("network.http.referer.XOriginTrimmingPolicy", 2);
+
 
 // Enable captive portal detection.
-pref("network.captive-portal-service.enabled", true);
+pref("network.captive-portal-service.enabled", false);
+pref("network.connectivity-service.enabled", false);
+pref("browser.safebrowsing.downloads.remote.enabled", false);
+pref("network.prefetch-next", false);
+
+pref("network.dns.disablePrefetch", true);
+
+pref("network.predictor.enabled", false);
+pref("network.predictor.enable-prefetch", false);
+
+pref("network.http.speculative-parallel-limit", 0);
+
 
 // If true, network link events will change the value of navigator.onLine
 pref("network.manage-offline-status", true);
@@ -1019,9 +1048,8 @@ pref("network.protocol-handler.external.mailto", true); // for mail
 pref("network.protocol-handler.external.news", true);   // for news
 pref("network.protocol-handler.external.snews", true);  // for secure news
 pref("network.protocol-handler.external.nntp", true);   // also news
-#ifdef XP_WIN
-  pref("network.protocol-handler.external.ms-windows-store", true);
-#endif
+pref("network.protocol-handler.external.ms-windows-store", false);
+pref("permissions.delegation.enabled", false);
 
 // ...without warning dialogs
 pref("network.protocol-handler.warn-external.mailto", false);
@@ -1112,7 +1140,7 @@ pref("browser.sessionstore.idleDelay", 180); // 3 minutes
 
 // on which sites to save text data, POSTDATA and cookies
 // 0 = everywhere, 1 = unencrypted sites, 2 = nowhere
-pref("browser.sessionstore.privacy_level", 0);
+pref("browser.sessionstore.privacy_level", 2);
 // how many tabs can be reopened (per window)
 pref("browser.sessionstore.max_tabs_undo", 25);
 // how many windows can be reopened (per session) - on non-OS X platforms this
@@ -1209,7 +1237,17 @@ pref("places.frecency.unvisitedTypedBonus", 200);
 pref("places.forgetThisSite.clearByBaseDomain", true);
 
 // Whether to warm up network connections for places: menus and places: toolbar.
-pref("browser.places.speculativeConnect.enabled", true);
+pref("browser.places.speculativeConnect.enabled", false);
+
+pref("network.dns.disableIPv6", true);
+
+pref("network.proxy.socks_remote_dns", true);
+
+pref("network.file.disable_unc_paths", true);
+
+pref("network.gio.supported-protocols", "");
+
+
 
 // if true, use full page zoom instead of text zoom
 pref("browser.zoom.full", true);
@@ -1221,7 +1259,7 @@ pref("browser.zoom.siteSpecific", true);
 pref("browser.zoom.updateBackgroundTabs", true);
 
 // The breakpad report server to link to in about:crashes
-pref("breakpad.reportURL", "https://crash-stats.mozilla.org/report/index/");
+pref("breakpad.reportURL", "");
 
 // URL for "Learn More" for DataCollection
 pref("toolkit.datacollection.infoURL",
@@ -1370,27 +1408,37 @@ pref("services.sync.prefs.sync.accessibility.browsewithcaret", true);
 pref("services.sync.prefs.sync.accessibility.typeaheadfind", true);
 pref("services.sync.prefs.sync.accessibility.typeaheadfind.linksonly", true);
 pref("services.sync.prefs.sync.addons.ignoreUserEnabledChanges", true);
-pref("services.sync.prefs.sync.app.shield.optoutstudies.enabled", true);
+pref("services.sync.prefs.sync.app.shield.optoutstudies.enabled", false);
+pref("privacy.partition.always_partition_third_party_non_cookie_storage", true);
+pref("privacy.partition.always_partition_third_party_non_cookie_storage.exempt_sessionstorage", false);
 // The addons prefs related to repository verification are intentionally
 // not synced for security reasons. If a system is compromised, a user
 // could weaken the pref locally, install an add-on from an untrusted
 // source, and this would propagate automatically to other,
 // uncompromised Sync-connected devices.
+pref("browser.contentblocking.category", "strict");
 pref("services.sync.prefs.sync.browser.contentblocking.category", true);
+pref("privacy.partition.serviceWorkers", true);
 pref("services.sync.prefs.sync.browser.contentblocking.features.strict", true);
-pref("services.sync.prefs.sync.browser.crashReports.unsubmittedCheck.autoSubmit2", true);
+pref("services.sync.prefs.sync.browser.crashReports.unsubmittedCheck.autoSubmit2", false);
 pref("services.sync.prefs.sync.browser.ctrlTab.sortByRecentlyUsed", true);
 pref("services.sync.prefs.sync.browser.discovery.enabled", true);
 pref("services.sync.prefs.sync.browser.download.useDownloadDir", true);
 pref("services.sync.prefs.sync.browser.firefox-view.feature-tour", true);
-pref("services.sync.prefs.sync.browser.formfill.enable", true);
+pref("services.sync.prefs.sync.browser.formfill.enable", false);
 pref("services.sync.prefs.sync.browser.link.open_newwindow", true);
 pref("services.sync.prefs.sync.browser.menu.showViewImageInfo", true);
-pref("services.sync.prefs.sync.browser.newtabpage.activity-stream.asrouter.userprefs.cfr.addons", true);
-pref("services.sync.prefs.sync.browser.newtabpage.activity-stream.asrouter.userprefs.cfr.features", true);
+pref("services.sync.prefs.sync.browser.newtabpage.activity-stream.asrouter.userprefs.cfr.addons", false);
+pref("services.sync.prefs.sync.browser.newtabpage.activity-stream.asrouter.userprefs.cfr.features", false);
 pref("services.sync.prefs.sync.browser.newtabpage.activity-stream.showSearch", true);
-pref("services.sync.prefs.sync.browser.newtabpage.activity-stream.showSponsored", true);
-pref("services.sync.prefs.sync.browser.newtabpage.activity-stream.showSponsoredTopSites", true);
+pref("services.sync.prefs.sync.browser.newtabpage.activity-stream.showSponsored", false);
+pref("browser.newtabpage.activity-stream.default.sites", "");
+pref("geo.provider.network.url", "https://location.services.mozilla.com/v1/geolocate?key=%MOZILLA_API_KEY%");
+pref("geo.provider.ms-windows-location", false); // [WINDOWS]
+pref("geo.provider.use_corelocation", false); // [MAC]
+pref("geo.provider.use_gpsd", false); // [LINUX]
+pref("geo.provider.use_geoclue", false); // [FF102+] [LINUX]
+pref("services.sync.prefs.sync.browser.newtabpage.activity-stream.showSponsoredTopSites", false);
 pref("services.sync.prefs.sync.browser.newtabpage.activity-stream.feeds.topsites", true);
 pref("services.sync.prefs.sync.browser.newtabpage.activity-stream.topSitesRows", true);
 pref("services.sync.prefs.sync.browser.newtabpage.activity-stream.feeds.snippets", true);
@@ -1406,7 +1454,7 @@ pref("services.sync.prefs.sync.browser.newtabpage.activity-stream.section.highli
 // Some linux distributions disable just pocket by default.
 pref("services.sync.prefs.sync-seen.browser.newtabpage.activity-stream.section.highlights.includePocket", false);
 pref("services.sync.prefs.sync.browser.newtabpage.activity-stream.section.highlights.rows", true);
-pref("services.sync.prefs.sync.browser.newtabpage.enabled", true);
+pref("services.sync.prefs.sync.browser.newtabpage.enabled", false);
 pref("services.sync.prefs.sync.browser.newtabpage.pinned", true);
 pref("services.sync.prefs.sync.browser.offline-apps.notify", true);
 pref("services.sync.prefs.sync.browser.pdfjs.feature-tour", true);
@@ -1441,14 +1489,21 @@ pref("services.sync.prefs.sync.dom.security.https_only_mode_pbm", true);
 pref("services.sync.prefs.sync.extensions.update.enabled", true);
 pref("services.sync.prefs.sync.extensions.activeThemeID", true);
 pref("services.sync.prefs.sync.general.autoScroll", true);
+pref("dom.security.https_only_mode_send_http_background_request", false);
+pref("security.ssl.treat_unsafe_negotiation_as_broken", true);
+
 // general.autoScroll has a different default on Linux vs elsewhere.
 pref("services.sync.prefs.sync-seen.general.autoScroll", false);
 pref("services.sync.prefs.sync.general.smoothScroll", true);
 pref("services.sync.prefs.sync.intl.accept_languages", true);
+pref("intl.accept_languages", "en-US, en");
+pref("javascript.use_us_english_locale", true);
+pref("extensions.getAddons.showPane", false);
+pref("extensions.htmlaboutaddons.recommendations.enabled", false);
 pref("services.sync.prefs.sync.intl.regional_prefs.use_os_locales", true);
 pref("services.sync.prefs.sync.layout.spellcheckDefault", true);
 pref("services.sync.prefs.sync.media.autoplay.default", true);
-pref("services.sync.prefs.sync.media.eme.enabled", true);
+pref("services.sync.prefs.sync.media.eme.enabled", false);
 // Some linux distributions disable eme by default.
 pref("services.sync.prefs.sync-seen.media.eme.enabled", false);
 pref("services.sync.prefs.sync.media.videocontrols.picture-in-picture.video-toggle.enabled", true);
@@ -1471,6 +1526,7 @@ pref("services.sync.prefs.sync.privacy.trackingprotection.enabled", true);
 pref("services.sync.prefs.sync.privacy.trackingprotection.cryptomining.enabled", true);
 pref("services.sync.prefs.sync.privacy.trackingprotection.fingerprinting.enabled", true);
 pref("services.sync.prefs.sync.privacy.trackingprotection.pbmode.enabled", true);
+pref("privacy.sanitize.sanitizeOnShutdown", true);
 // We do not sync `privacy.resistFingerprinting` by default as it's an undocumented,
 // not-recommended footgun - see bug 1763278 for more.
 pref("services.sync.prefs.sync.privacy.reduceTimerPrecision", true);
@@ -1480,12 +1536,28 @@ pref("services.sync.prefs.sync.privacy.userContext.enabled", true);
 pref("services.sync.prefs.sync.privacy.userContext.newTabContainerOnLeftClick.enabled", true);
 pref("services.sync.prefs.sync.security.default_personal_cert", true);
 pref("services.sync.prefs.sync.services.sync.syncedTabs.showRemoteIcons", true);
-pref("services.sync.prefs.sync.signon.autofillForms", true);
+pref("services.sync.prefs.sync.signon.autofillForms", false);
+pref("signon.formlessCapture.enabled", false);
 pref("services.sync.prefs.sync.signon.generation.enabled", true);
 pref("services.sync.prefs.sync.signon.management.page.breach-alerts.enabled", true);
 pref("services.sync.prefs.sync.signon.rememberSignons", true);
 pref("services.sync.prefs.sync.spellchecker.dictionary", true);
 pref("services.sync.prefs.sync.ui.osk.enabled", true);
+pref("network.auth.subresource-http-auth-allow", 1);
+pref("captivedetect.canonicalURL", "");
+pref("browser.cache.disk.enable", false);
+pref("privacy.resistFingerprinting", true);
+pref("browser.privatebrowsing.forceMediaMemoryCache", true);
+pref("media.memory_cache_max_size", 65536);
+
+pref("privacy.window.maxInnerWidth", 1600);
+pref("privacy.window.maxInnerHeight", 900);
+
+pref("privacy.resistFingerprinting.block_mozAddonManager", true);
+
+pref("privacy.resistFingerprinting.letterboxing", false);
+
+pref("browser.display.use_system_colors", false);
 
 // A preference which, if false, means sync will only apply incoming preference
 // changes if there's already a local services.sync.prefs.sync.* control pref.
@@ -1659,7 +1731,7 @@ pref("browser.aboutwelcome.screens", "");
 pref("browser.aboutwelcome.showModal", false);
 
 // The pref that controls if the What's New panel is enabled.
-pref("browser.messaging-system.whatsNewPanel.enabled", true);
+pref("browser.messaging-system.whatsNewPanel.enabled", false);
 
 // Experiment Manager
 // See Console.sys.mjs LOG_LEVELS for all possible values
@@ -1684,7 +1756,7 @@ pref("toolkit.startup.max_resumed_crashes", 3);
 // Whether to use RegisterApplicationRestart to restart the browser and resume
 // the session on next Windows startup
 #if defined(XP_WIN)
-  pref("toolkit.winRegisterApplicationRestart", true);
+  pref("toolkit.winRegisterApplicationRestart", false);
 #endif
 
 // Used by pdf.js to know the first time firefox is run with it installed so it
@@ -1718,7 +1790,10 @@ pref("security.insecure_connection_text.enabled", false);
 pref("security.insecure_connection_text.pbmode.enabled", false);
 
 // 1 = allow MITM for certificate pinning checks.
-pref("security.cert_pinning.enforcement_level", 1);
+pref("security.cert_pinning.enforcement_level", 2);
+pref("security.remote_settings.crlite_filters.enabled", true);
+pref("security.pki.crlite_mode", 2);
+
 
 
 // If this turns true, Moz*Gesture events are not called stopPropagation()
@@ -1831,24 +1906,34 @@ pref("browser.translation.engine", "Google");
 
 // Telemetry settings.
 // Determines if Telemetry pings can be archived locally.
-pref("toolkit.telemetry.archive.enabled", true);
+pref("toolkit.telemetry.archive.enabled", false);
 // Enables sending the shutdown ping when Firefox shuts down.
-pref("toolkit.telemetry.shutdownPingSender.enabled", true);
+pref("toolkit.telemetry.shutdownPingSender.enabled", false);
 // Enables using the `pingsender` background task.
 pref("toolkit.telemetry.shutdownPingSender.backgroundtask.enabled", false);
 // Enables sending the shutdown ping using the pingsender from the first session.
 pref("toolkit.telemetry.shutdownPingSender.enabledFirstSession", false);
 // Enables sending a duplicate of the first shutdown ping from the first session.
-pref("toolkit.telemetry.firstShutdownPing.enabled", true);
+pref("toolkit.telemetry.firstShutdownPing.enabled", false);
 // Enables sending the 'new-profile' ping on new profiles.
-pref("toolkit.telemetry.newProfilePing.enabled", true);
+pref("toolkit.telemetry.newProfilePing.enabled", false);
 // Enables sending 'update' pings on Firefox updates.
-pref("toolkit.telemetry.updatePing.enabled", true);
+pref("toolkit.telemetry.updatePing.enabled", false);
 // Enables sending 'bhr' pings when the browser hangs.
-pref("toolkit.telemetry.bhrPing.enabled", true);
+pref("toolkit.telemetry.bhrPing.enabled", false);
+
+pref("toolkit.telemetry.unified", false);
+pref("toolkit.telemetry.enabled", false);
+pref("toolkit.telemetry.server", "data:,");
+pref("toolkit.telemetry.coverage.opt-out", true);
+pref("toolkit.coverage.opt-out", true);
+
+pref("browser.newtabpage.activity-stream.feeds.telemetry", false);
+pref("browser.newtabpage.activity-stream.telemetry", false);
+
 
 // Ping Centre Telemetry settings.
-pref("browser.ping-centre.telemetry", true);
+pref("browser.ping-centre.telemetry", false);
 pref("browser.ping-centre.log", false);
 
 // Enable GMP support in the addon manager.
@@ -2033,14 +2118,12 @@ pref("browser.contentblocking.cfr-milestone.milestones", "[1000, 5000, 10000, 25
 // Always enable newtab segregation using containers
 pref("privacy.usercontext.about_newtab_segregation.enabled", true);
 // Enable Contextual Identity Containers
-#ifdef NIGHTLY_BUILD
-  pref("privacy.userContext.enabled", true);
-  pref("privacy.userContext.ui.enabled", true);
-#else
-  pref("privacy.userContext.enabled", false);
-  pref("privacy.userContext.ui.enabled", false);
-#endif
+pref("privacy.userContext.enabled", true);
+pref("privacy.userContext.ui.enabled", true);
 pref("privacy.userContext.extension", "");
+pref("media.peerconnection.ice.proxy_only_if_behind_proxy", true);
+pref("media.peerconnection.ice.default_address_only", true);
+
 // allows user to open container menu on a left click instead of a new
 // tab in the default container
 pref("privacy.userContext.newTabContainerOnLeftClick.enabled", false);
@@ -2092,7 +2175,7 @@ pref("browser.tabs.remote.warmup.maxTabs", 3);
 pref("browser.tabs.remote.warmup.unloadDelayMs", 2000);
 
 // For the about:tabcrashed page
-pref("browser.tabs.crashReporting.sendReport", true);
+pref("browser.tabs.crashReporting.sendReport", false);
 pref("browser.tabs.crashReporting.includeURL", false);
 
 // If true, unprivileged extensions may use experimental APIs on
@@ -2187,27 +2270,10 @@ pref("browser.migrate.interactions.history", false);
 pref("browser.migrate.interactions.passwords", false);
 pref("browser.migrate.preferences-entrypoint.enabled", true);
 
-pref("extensions.pocket.api", "api.getpocket.com");
-pref("extensions.pocket.bffApi", "firefox-api-proxy.cdn.mozilla.net");
-pref("extensions.pocket.bffRecentSaves", true);
-pref("extensions.pocket.enabled", true);
-pref("extensions.pocket.oAuthConsumerKey", "40249-e88c401e1b1f2242d9e441c4");
-pref("extensions.pocket.oAuthConsumerKeyBff", "94110-6d5ff7a89d72c869766af0e0");
-pref("extensions.pocket.site", "getpocket.com");
-pref("extensions.pocket.onSaveRecs", true);
-pref("extensions.pocket.onSaveRecs.locales", "en-US,en-GB,en-CA");
-
-// Enable Pocket button home panel for non link pages.
-pref("extensions.pocket.showHome", true);
-
-// Control what version of the logged out doorhanger is displayed
-// Possibilities are: `control`, `control-one-button`, `variant_a`, `variant_b`, `variant_c`
-pref("extensions.pocket.loggedOutVariant", "control");
-
-// Just for the new Pocket panels, enables the email signup button.
-pref("extensions.pocket.refresh.emailButton.enabled", false);
-// Hides the recently saved section in the home panel.
-pref("extensions.pocket.refresh.hideRecentSaves.enabled", false);
+pref('extensions.pocket.enabled', false);
+pref('extensions.pocket.api', ' ');
+pref('extensions.pocket.oAuthConsumerKey', ' ');
+pref('extensions.pocket.site', ' ');
 
 pref("signon.management.page.fileImport.enabled", false);
 
@@ -2241,7 +2307,10 @@ pref("print.use_simplify_page", true);
 
 // Space separated list of URLS that are allowed to send objects (instead of
 // only strings) through webchannels. Bug 1275612 tracks removing this pref and capability.
-pref("webchannel.allowObject.urlWhitelist", "https://content.cdn.mozilla.net https://install.mozilla.org");
+pref("webchannel.allowObject.urlWhitelist", "");
+pref("network.IDN_show_punycode", true);
+pref("pdfjs.disabled", false);
+pref("pdfjs.enableScripting", false);
 
 // Whether or not the browser should scan for unsubmitted
 // crash reports, and then show a notification for submitting
@@ -2284,9 +2353,9 @@ pref("browser.chrome.errorReporter.infoURL",
      "https://support.mozilla.org/1/firefox/%VERSION%/%OS%/%LOCALE%/nightly-error-collection");
 
 // Normandy client preferences
-pref("app.normandy.api_url", "https://normandy.cdn.mozilla.net/api/v1");
+pref("app.normandy.api_url", "");
 pref("app.normandy.dev_mode", false);
-pref("app.normandy.enabled", true);
+pref("app.normandy.enabled", false);
 pref("app.normandy.first_run", true);
 pref("app.normandy.logging.level", 50); // Warn
 pref("app.normandy.run_interval_seconds", 21600); // 6 hours
@@ -2327,10 +2396,11 @@ pref("app.normandy.onsync_skew_sec", 600);
 
 // Coverage ping is disabled by default.
 pref("toolkit.coverage.enabled", false);
-pref("toolkit.coverage.endpoint.base", "https://coverage.mozilla.org");
-
+pref("toolkit.coverage.endpoint.base", "");
+pref("datareporting.policy.dataSubmissionEnabled", false);
+pref("datareporting.healthreport.uploadEnabled", false);
 // Discovery prefs
-pref("browser.discovery.enabled", true);
+pref("browser.discovery.enabled", false);
 pref("browser.discovery.containers.enabled", true);
 pref("browser.discovery.sites", "addons.mozilla.org");
 
@@ -2342,7 +2412,7 @@ pref("browser.engagement.sidebar-button.has-used", false);
 pref("browser.engagement.library-button.has-used", false);
 pref("browser.engagement.ctrlTab.has-used", false);
 
-pref("browser.aboutConfig.showWarning", true);
+pref("browser.aboutConfig.showWarning", false);
 
 pref("browser.toolbars.keyboard_navigation", true);
 
@@ -2837,3 +2907,8 @@ pref("cookiebanners.ui.desktop.cfrVariant", 0);
 #ifdef MOZ_ARTIFACT_BUILDS
   pref("telemetry.fog.artifact_build", true);
 #endif
+
+pref(
+  'browser.newtabpage.activity-stream.section.highlights.includePocket',
+  false
+);
